snuba/datasets/transactions.py

```diff
@@ -147,5 +147,12 @@ def get_extensions(self) -> Mapping[str, QueryExtension]:
             ),
         }
 
+    def column_expr(self, column_name, body):
+        if column_name == 'ip_address_v4':
```

Comment:
Please add a comment that this should be removed once we get at least to version 0.0.19 of clickhouse driver because in 0.0.18 it does not support ip addresses.